File: proxy/tests/script_engine/body_set.lua
pcall(require, "../../script_libs/lua/roxy.lua")
---@type Extension
local body_set = {
	request = function(flow)
		flow.request.body.text = "rewrite request"
	end,
	response = function(flow)
		flow.response.body.text = "rewrite response"
	end,
}
Extensions = { body_set }

File: proxy/tests/script_engine/body_set.js
/// <reference path="../../script_libs/js/index.d.ts" />
/** @type {Extension} */
const bodySet = {
  request(flow) {
    flow.request.body.text = "rewrite request";
  },
  response(flow) {
    flow.response.body.text = "rewrite response";
  }
};
globalThis.extensions = [bodySet];
